src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+import { ADD_BOOK, REMOVE_BOOK } from '../constants';
```

Comment:
- [OPTIONAL] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK` Kindly address this 👍🏻

Reply:
I followed this in `constants.js`. Can you have a look at here? https://github.com/PrangonGhose/bookstore-with-react/blob/145776b7923ccf3e2ec09973c2d946539edec2c4/src/redux/constants.js#L1-L2

Reply:
- [ ] Soory for late reply, I can see that I'm made you being  late 🙏🏻 🙏🏻  sorry my freind I was not around, please submit the project the next code reviewer will approve this 👍🏻